CardGame.playground/Contents.swift

```diff
@@ -2,41 +2,115 @@ import Foundation
 
 //: ## Step 1
 //: Create an enumeration for the value of a playing card. The values are: `ace`, `two`, `three`, `four`, `five`, `six`, `seven`, `eight`, `nine`, `ten`, `jack`, `queen`, and `king`. Set the raw type of the enum to `Int` and assign the ace a value of `1`.
-
-
+enum PlayingCard: Int {
+    
+    case ace = 1
```

Comment:
Since you gave this a default raw value of 1, case two will be automatically assigned a raw value of 2, case three a raw value of 3. So on and so forth. Just a tip so you can save yourself some time.

Reply:
Swift